tests/bindings/nodejs/HawkTracerClient.test.ts

```diff
@@ -72,50 +72,74 @@ test("Adds klass_name", (done) => {
     });
 })
 
+
+test("start (resolve)", () => {
+    expect.assertions(1);
+    mockNativeClient.start = jest.fn().mockReturnValue(true);
+    const client = new HawkTracerClient("abc");
+    return client.start().then(data => expect(data).toEqual(client));
+});
+
+test("start (reject)", () => {
+    expect.assertions(1);
+    mockNativeClient.start = jest.fn().mockImplementationOnce(() => {
+        throw new Error();
+    });
+    const client = new HawkTracerClient("abc");
+    client.start().catch(e => expect(e).toBeInstanceOf(Error))
+});
+
+test("start (retries)", () => {
+    expect.assertions(1);
+    mockNativeClient.start = jest.fn().mockReturnValue(false);
+    const client = new HawkTracerClient("abc", 0);
```

Comment:
We could also check `start()` is called the correct times.

Reply:
That's a wise suggestion

Reply:
This is done now